config/settings.yml

```diff
@@ -91,6 +91,7 @@ features:
   funding_uploads: true
   register_api: true
   lead_partners: false
+  itt_reform: false
```

Comment:
Can you add a `true` for the `review.yml`
So we can all marvel at your awesome work

:pray: